CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED] neptune 1.9.2
+
+## Changes
+- Removed most of the default values for function parameters in `searching_entries.py` ([#1653](https://github.com/neptune-ai/neptune-client/pull/1653))
```

Comment:
That's internal change so there is no need to be included in CHANGELOG :wink:

Reply:
Done